aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-vb.md

```diff
@@ -176,7 +176,7 @@ Let's augment the `RolesUserList` GridView so that it includes a column of "Remo
 
 Start by adding a Delete button field to the GridView. Make this field appear as the left most filed and change its `DeleteText` property from "Delete" (the default) to "Remove".
 
-[![Add the](assigning-roles-to-users-vb/_static/image20.png)](assigning-roles-to-users-vb/_static/image19.png)
+[![Add a Delete button.](assigning-roles-to-users-vb/_static/image20.png)](assigning-roles-to-users-vb/_static/image19.png)
```

Comment:
Let's make it a little more descriptive. Something like "Screenshot that shows how to add the “Remove” button in the Fields window."